backend/src/main/java/com/jujeol/drink/application/RecommendFactory.java

```diff
@@ -18,10 +19,10 @@
     private final Map<MemberStatus, RecommendStrategy> recommendStrategyMap;
 
     public RecommendFactory(RecommendationSystem recommendationSystem,
-            DrinkRepository drinkRepository) {
+            DrinkRepository drinkRepository, PreferenceRepository preferenceRepository) {
```

Comment:
```suggestion
            DrinkRepository drinkRepository, 
            PreferenceRepository preferenceRepository
            ) {
```
이런 느낌으로 파라미터 정리하면 눈에 더 잘 들어올 거 같아요!